README.md

```diff
@@ -1,61 +1,46 @@
-# Azure Functions Serverless Plugin 
+# Azure Functions Serverless Plugin
 
 This plugin enables Azure Functions support within the Serverless Framework.
 
 [![Coverage Status](https://coveralls.io/repos/github/serverless/serverless-azure-functions/badge.svg)](https://coveralls.io/github/serverless/serverless-azure-functions)
 
-## Getting started
+## Quickstart
 
 ### Pre-requisites
 
 1. Node.js v6.5.0+ *(this is the runtime version supported by Azure Functions)*
 2. Serverless CLI `v1.9.0+`. You can run `npm i -g serverless` if you don't already have it.
 3. An Azure account. If you don't already have one, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/) that includes $200 of free credit.
 
```

Comment:
Azure plugin that allows you to work with Azure Functions
npm install -g serverless-azure-functions

Reply:
Because that's in the `devDependencies`, the `npm install` should take care of it.